rasterio/rio/pyramid.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+"""Manage overviews of a dataset."""
+
+import logging
+
+import click
+
+import rasterio
+from rasterio.enums import Resampling
+
+from . import options
+
+
+def build_handler(ctx, param, value):
+    if value:
+        try:
+            if '^' in value:
+                base, exp_range = value.split('^')
+                exp_min, exp_max = (int(v) for v in exp_range.split('..'))
+                value = [pow(int(base), k) for k in range(exp_min, exp_max+1)]
+            else:
```

Comment:
@sgillies lines 21-24 are showing up as missing coverage; Coveralls just isn't showing this correctly at the moment.